azure-pipelines.yml

```diff
@@ -0,0 +1,27 @@
+jobs:
+- job: BuildLinuxWithSonarCloud
+  displayName: Build for Linux\CMake With SonarCloud
+  pool:
+    vmImage: 'ubuntu-latest'
+  variables:
+    BUILD_WRAPPER_DOWNLOAD_URL: https://sonarcloud.io/static/cpp/build-wrapper-linux-x86.zip
+  steps:
+  - task: SonarCloudPrepare@1
+    inputs:
+      SonarCloud: 'SonarCloud'
```

Comment:
I would add a comment, something like "name of the SonarCloud service connection you created"

Reply:
Indeed. Added.